### PR TITLE
Add title tag to DuckDuckHack header.

### PR DESCRIPTION
**Pull request by jbarrett**
Missing title reported by @MrChrisW - thanks.

@sdougbrown @chrismorast Would you like me to change the text?

### CODE REVIEW

**Comment by MrChrisW** [comment]
Afternoon @jbarrett At least I know were the repository is now, thanks :laughing:

**Comment by sdougbrown** [comment]
@zekiel - this is kind of minor but if you'd like adjust the wording please by all means step in here. :)

File: share/site/duckduckhack/inc/head.tx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<title><: lp("title","DuckDuckHack - an open source community for DuckDuckGo Instant Answers") :></title>
```
Comment (nilnilnil) [inline share/site/duckduckhack/inc/head.tx]: Is this the real verbiage? Isn't that too long for the tab? `DuckDuckHack - Instant Answers`?

Comment (bsstoner) [inline share/site/duckduckhack/inc/head.tx]: I like the long one. Doesn't really matter if it overflows the tab, main purpose for being descriptive is in search results.

**Comment by sdougbrown** [comment]
Yeah I'm doing some SEO here, haha

**Comment by zekiel** [comment]
lgtm

**Comment by nilnilnil** [comment]
:+1:

**Pull request merged**